### ISSUE TITLE
[Bug]: Broken running detection on installer (version >= 4.4.8 Windows)

### ISSUE THREAD

**Issue by AngusMcGyver** [BelledonneCommunications/linphone-desktop#695]
### Context

It seams the application running detection of the installer is somehow broken since 4.4.8.

I'm trying to upgrade to the latest version 4.4.9 from 4.4.7 but the installer exits with the following:
```
Linphone is running. Please close it first from the application menu and restart the installation.
```
So i uninstalled the application and reboot the system. Same error message and the installer won't start. Same for the installer of 4.4.8.

The installer for 4.4.7 works fine.

Then i also tried to run the installer inside a VirtualBox VM, where linphone was never installed before, same behavior.

### General information

- Device: Lenovo T14
- OS: Windows 10 21H2
- Version of the App: [4.4.8] & [4.4.9]

### Expected behaviour

The Installer of 4.4.8 and 4.4.9 should works as the versions before.

### To Reproduce

1. Download the installer from https://download.linphone.org/releases/windows/app/
2. Run it.
3. Get the error message.

**Comment by julonexus** [comment]
As long as you have a windows titled "Linphone", it will consider that the app is still running.

**Comment by AngusMcGyver** [comment]
you are right… , thanks for clearing this up.
the folder i downloaded the installer in is called `linphone`. so the explorer window i start the installer from is called linphone. after change the folder name i can run the installer just fine. :facepalm:
inside the virtual box its the same issue, i just mounted my software folder into the vm.

imho a terrible way to implement such a check. would be better to search the process list for `linphone.exe`.
at least the error message needs an update.

**Comment by julonexus** [comment]
Yes, I agree. NSIS is very limited without plugins.
This detection will be available for 4.5.
